public/index.html

```diff
@@ -1,43 +1,19 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
```

Comment:
I think head should be indented. Were you able to set up Prettier?

Reply:
^ Yes, I believe that head should be indented as well. Let me know you need help setting up and using Prettier.